### PR TITLE
eNikshay Agency Registration

### PR DESCRIPTION
**Pull request by esoergel**
https://docs.google.com/spreadsheets/d/1R1rv3bpJxPRQSeTWDMU3IXX4lVhWMfZvLnxye3riizQ/edit#gid=1458284637

### CODE REVIEW

**Comment by esoergel** [comment]
@proteusvacuum @orangejenny

File: custom/enikshay/user_setup.py [hunk 1]
```diff
@@ -267,50 +269,104 @@ def __init__(self, *args, **kwargs):
...
 
-class ENikshayUserDataEditor(CustomDataEditor):
-    """Custom User Data (everywhere it turns up)"""
-    def __init__(self, *args, **kwargs):
-        super(ENikshayUserDataEditor, self).__init__(*args, **kwargs)
+# pcp -> MBBS
+# pac -> AYUSH/other
+# plc -> Private Lab
+# pcc -> pharmacy / chemist
+# dto -> Field officer??
```
Comment (nickpell) [inline custom/enikshay/user_setup.py]: I think field officer is pdr?

**Review by nickpell — APPROVED** [review]

File: custom/enikshay/user_setup.py (hunk 1 above)
Comment (esoergel) [inline custom/enikshay/user_setup.py]: Nice, thanks - [the spec](https://docs.google.com/spreadsheets/d/1R1rv3bpJxPRQSeTWDMU3IXX4lVhWMfZvLnxye3riizQ/edit#gid=1458284637) said dto, but seemed unsure

Comment (proteusvacuum) [inline custom/enikshay/user_setup.py]: that column in the spec was detailing the location type that should be created when someone selects that particular user type. For the field officer, I don't think they get a new location created, but instead are assigned to the DTO level (AFAIK, I think we were going to check this with Erin, which is why I put it in parens). 

I don't know where in this code that happens, if at all, so I'm assuming this is resolved.

**Pull request merged**